webpack/components/PreupgradeReports/PreupgradeReports.js

```diff
@@ -29,22 +32,38 @@ const PreupgradeReports = ({
     );
   }
 
+  const isSelected = entry => checked.some(item => item.id === entry.id);
+
+  const anySelected = checked.length > 0;
+
+  const toggleSelected = (entry, isEntrySelected) => {
+    if (isEntrySelected) {
+      setChecked(checked.filter(item => item.id !== entry.id));
+    } else {
+      setChecked([entry, ...checked]);
+    }
+  };
+
   return (
     <LoadingState loading={loading}>
       <Row>
         <div className="title-filter col-md-4">&nbsp;</div>
         <div id="title_action" className="col-md-8">
           <div className="btn-toolbar pull-right">
-            <FixAllButton
```

Comment:
Is the "Fix all" button gone for good? Meaning if someone wants to fix everything, they will have to select every single checkbox on the page?

Reply:
Once we have a list header, there will be a checkbox in the header that selects all - same as we currently have on hosts page. That way, 'Fix all' will be performed with 2 clicks.

We could keep the 'Fix all' button if you prefer, but it completely ignores the selection and I am worried it might confuse some users.

Reply:
Then it is fine